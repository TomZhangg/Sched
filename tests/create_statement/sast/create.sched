#( Create a schedule that is named with schedule kind day and no inline item options )#
Create Schedule Day trump_nov_7;
