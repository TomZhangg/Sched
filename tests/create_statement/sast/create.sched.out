SCS(
  SSchedule(
    SNamed(
      <sched-kind>: Day
      None
      typ: CId, sx: SId(trump_nov_7)
    )
  )
)
